packages/office-ui-fabric-react/src/utilities/color/getColorFromHSV.ts

```diff
@@ -0,0 +1,23 @@
+import { IHSV, IColor } from './interfaces';
+import { MAX_COLOR_ALPHA } from './consts';
+import { hsv2rgb } from './hsv2rgb';
+import { hsv2hex } from './hsv2hex';
+import { _rgbaOrHexString } from './_rgbaOrHexString';
+
+/**
+ * Converts an HSV color (and optional alpha value) to a color object.
+ * If `a` is not given, a default of 100 is used.
+ * Hex in the returned value will *not* be prefixed with #.
+ * If `a` is unspecified or 100, the result's `str` property will contain a hex value
+ * (*not* prefixed with #)
+ */
+export function getColorFromHSV(hsv: IHSV, a?: number): IColor {
+  const { h, s, v } = hsv;
+  a = typeof a === 'number' ? a : MAX_COLOR_ALPHA;
```

Comment:
It seems like this logic should be in the _rgbaOrHexString, and since you're doing this here the hex value you generate below is never used in _rgbaOrHexString which seems weird to pass the hex value through.

Reply:
All this code is copied and pasted, all I did is move it. Improvements can/should be made in a subsequent PR.